flink-connectors/flink-connector-files/src/main/java/org/apache/flink/connector/file/table/PartitionCommitPolicyFactory.java

```diff
@@ -63,11 +75,23 @@ public List<PartitionCommitPolicy> createPolicyChain(
                                             successFileName, fsSupplier.get());
                                 case PartitionCommitPolicy.CUSTOM:
                                     try {
-                                        return (PartitionCommitPolicy)
-                                                cl.loadClass(customClass).newInstance();
-                                    } catch (ClassNotFoundException
-                                            | IllegalAccessException
-                                            | InstantiationException e) {
+                                        if (parameters != null && !parameters.isEmpty()) {
```

Comment:
I'm wondering if change it to 
```
parameters != null
```
Can it also get the constructor with empty arguments?
If so, we should consider to change to `parameters != null` to make code more clear.

Reply:
If users input
`'sink.partition-commit.policy.class.parameters'='',`
parameters != null but parameters will empty,
so yes, it maybe get the constructor with empty arguments.

Reply:
> I'm wondering if change it to
> 
> ```
> parameters != null
> ```
> 
> Can it also get the constructor with empty arguments? If so, we should consider to change to `parameters != null` to make code more clear.

If users input
`'sink.partition-commit.policy.class.parameters'='',`
parameters != null but parameters will empty,
so yes, it maybe get the constructor with empty arguments.